.circleci/config.yml

```diff
@@ -19,14 +19,7 @@ jobs:
   release:
     executor: gusto/ruby-2-3
     steps:
-      - gusto/bundle-install
-      - gusto/yarn-install
-      - run:
-          command: |
-            mkdir -p ~/.gem
-            echo -e "---\r\n:rubygems_api_key: $GEM_HOST_API_KEY" > ~/.gem/credentials
-            chmod 0600 /home/circleci/.gem/credentials
```

Comment:
And this third command also looks like we should keep it around

Reply:
These were all rolled up into a shared command here: https://github.com/Gusto/circleci-orbs/blob/master/orbs/gusto/commands/semantic-release-ruby.yml